test/compflags/kushal/llvm-incremental.bad

```diff
@@ -0,0 +1 @@
+Incremental compilation is not yet supported with LLVM
```

Comment:
Actually I don't have llvm setup , so when I run the test ,all I get is ```This compiler was built without LLVM support``` (without any line number or file number (although a USR_FATAL warning) , so I thought it should also be something like this. However I am not sure on how to test this